src/test/java/com/carmen/app/control/CarServiceTest.java

```diff
@@ -62,60 +68,62 @@ public void testCarCreated() {
 
 	@Test
 	public void testGetCars() {
-		when(this.persistenceService.getAll(Mockito.eq(Car.class),Mockito.anyMap(),Mockito.anyString())).thenReturn(query);
 		List<Car> expectedCars = new ArrayList<Car>();
-		List<Car> cars = this.carService.getCars(1 , 2, "Seat", "brand");
+		when(this.persistenceService.getAll(Mockito.eq(Car.class),Mockito.anyString(),Mockito.anyInt(),Mockito.anyInt(), Mockito.anyString(), Mockito.anyString())).thenReturn(expectedCars);
```

Comment:
Test is not really very usefull since the list you return is empty, would be a better case when returning a list of cars with some elements and then you can check list length, elements, order...etc.
An empty list is also one case to test, but there are many more cases.